ex7.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Include a shebang (#!/bin/bash) at the beginning of the script: Although you mentioned it in the comments, it's a good practice to include the shebang line as the first line of the script. This ensures the script is executed using the correct interpreter.